.gitignore

```diff
@@ -27,6 +27,8 @@ id_rsa*
 hosts*
 *.flag
 keys.txt
+results/
+module.json
```

Comment:
The module.json files are only needed for generating terraform documentation in the README.md files. The module.json files do not need to exist in the repo.